docs/pages/use-cases/nftqueries/nft4.mdx

```diff
@@ -0,0 +1,200 @@
+
+- UTXOs: The Unspent Transaction Outputs associated with the wallet that will fund the minting transaction.
+
+- Slot/Block Tip: The current slot or block tip to ensure the transaction is valid within the blockchain context.
+
+### 4.1 Protocol parameters
+
+```sql
+        SELECT 
+		    ep.epoch_no AS epoch,
+		    ep.min_fee_a,
+		    ep.min_fee_b,
+		    ep.max_block_size,
+		    ep.max_tx_size,
+		    ep.max_bh_size as max_block_header_size,
+		    ep.key_deposit,
+		    ep.pool_deposit,
+		    ep.max_epoch as e_max,
+		    ep.optimal_pool_count as n_opt,
+		    ep.influence as a0,
+		    ep.monetary_expand_rate as rho,
+		    ep.treasury_growth_rate as tau,
+		    ep.decentralisation as decentralisation_param,
+		    ep.extra_entropy,
+		    ep.protocol_major AS protocol_major_ver,
+		    ep.protocol_minor AS protocol_minor_ver,
+		    ep.min_utxo_value AS min_utxo,
+		    ep.min_pool_cost,
+		    ep.nonce,
+		    cm.costs AS cost_models,
+		    ep.price_mem,
+		    ep.price_step,
+		    ep.max_tx_ex_mem,
+		    ep.max_tx_ex_steps,
+		    ep.max_block_ex_mem,
+		    ep.max_block_ex_steps,
+		    ep.max_val_size,
+		    ep.collateral_percent,
+		    ep.max_collateral_inputs,
+		    ep.coins_per_utxo_size,
+		  	ep.coins_per_utxo_size as coins_per_utxo_word
+		FROM 
+		    epoch_param ep
+		LEFT JOIN cost_model cm ON cm.id = ep.cost_model_id
+		ORDER BY epoch_no DESC
+		LIMIT 1
+
+```
+
+<details>
+<summary>
+Expected results format
+</summary>
+
+```sql
+
+epoch | min_fee_a | min_fee_b | max_block_size | max_tx_size | max_block_header_size | key_deposit | .....
+------+-----------+-----------+----------------+-------------+-----------------------+-------------+------
+505   | 44        | 155,381   | 90,112	       | 16,384	     | 1,100	             | 2,000,000   | ....
+
+(Note: results trimmed for readability)             
+```
+</details>
+
+🔎 Looking a little closer, this query reads data from database, without modifying the blockchain itself. It retrieves information about the most recent epoch.
+
+`SELECT` lists the specific columns to retrieve from the database tables.
+
+`FROM epoch_param ep` specifies the main table to query, [epoch_param](https://github.com/cardano-foundation/cf-ledger-sync/blob/main/docs/pages/schema.md#epoch_param).
+
+`LEFT JOIN cost_model cm ON cm.id = ep.cost_model_id` joins data from the [cost_model](https://github.com/cardano-foundation/cf-ledger-sync/blob/main/docs/pages/schema.md#cost_model) table based on matching id values.
+`ORDER BY epoch_no DESC` sorts the results by `epoch_no` in descending order, ie. most recent epoch first.
+`LIMIT 1` restricts the output to only the first row, ie. the most recent epoch's data.
+
+### 4.2 UTXOs associated with specific address 
+
+For this query, we are concerned with this address `addr1qyvl8lmwpfq7pahuadpjwul9xlj7c0atr5hvu4r3cjsjlfua276aqezvwjsfucvq76welxmytfyt7ug5835wntzzdvgsyxh7q9`
```

Comment:
Maybe we can rephrase it? 
Let's use  address `addr1qyvl8lmwpfq7pahuadpjwul9xlj7c0atr5hvu4r3cjsjlfua276aqezvwjsfucvq76welxmytfyt7ug5835wntzzdvgsyxh7q9` as an example for this query